src/main/java/core/basesyntax/dao/jdbc/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,142 @@
+package core.basesyntax.dao.jdbc;
+
+import core.basesyntax.dao.ManufacturerDao;
+import core.basesyntax.exception.DataProcessingException;
+import core.basesyntax.lib.Dao;
+import core.basesyntax.model.Manufacturer;
+import core.basesyntax.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String queryInsert = "INSERT INTO manufacturers "
+                + "(manufacturer_name, manufacturer_country) "
+                + "VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement insert = connection.prepareStatement(queryInsert,
```

Comment:
let's move an opening of a statement inside of a try with resources construction and it will be closed if an exception occurs as well